bundle/src/main/java/com/adobe/acs/tools/explain_query/impl/ExplainQueryServlet.java

```diff
@@ -101,22 +101,26 @@
 
     private static final String QUERY_BUILDER = "queryBuilder";
 
-    private static final String[] LANGUAGES = new String[]{ SQL, SQL2, XPATH };
+    private static final String[] LANGUAGES = new String[]{SQL, SQL2, XPATH};
+
+    private static final Pattern PROPERTY_INDEX_PATTERN =
+            Pattern.compile("\\/\\*\\sproperty\\s([^\\s=]+)[=\\s]");
 
-    private static final Pattern PROPERTY_INDEX_PATTERN = Pattern.compile("\\/\\*\\sproperty\\s([^\\s=]+)[=\\s]");
-    private static final Pattern FILTER_PATTERN = Pattern.compile("\\[[^\\s]+\\]\\sas\\s\\[[^\\s]+\\]\\s\\/\\*\\sFilter\\(");
+    private static final Pattern FILTER_PATTERN =
+            Pattern.compile("\\[[^\\s]+\\]\\sas\\s\\[[^\\s]+\\]\\s\\/\\*\\sFilter\\(");
 
     @Property(label = "Query Logger Names",
-            description = "Logger names from which logs need to be collected while a query is executed",
+            description = "Logger names from which logs need to be collected while a query is executed. "
+                    + "Provide in the format '<package-name>=<mdc-filter-name>' where <mdc-filter-name>' is optional.",
             unbounded = PropertyUnbounded.ARRAY,
             value = {
-                    "org.apache.jackrabbit.oak.query",
-                    "org.apache.jackrabbit.oak.plugins.index"
+                    "org.apache.jackrabbit.oak.query=" + OAK_QUERY_ANALYZE,
```

Comment:
Updated the logger configs to they pair with optional filter values; this allows the oak to have filters and querybuilder to not have any.